etc/nanomq_dds_gateway.conf

```diff
@@ -0,0 +1,99 @@
+# Example1. (default)
+# This configure will make msgs with topic topic1 be shared in DDS network and MQTT network.
+# It receive messages from MQTT network in topic DDSCMD/topic1 and send them to DDS network with topic MQTT/topic1.
+# It receive messages from DDS network in topic MQTTCMD/topic1 and send them to MQTT network with topic DDS/topic1. 
+
+dds {
+	idl_type  = topic1Type
+	domain_id = 0
+}
+
+mqtt {
+	connector {
+        # # Bridge address: host:port .
+        # #
+        # # Value: String
+        # # Example: mqtt-tcp://127.0.0.1:1883
+        # #          tls+mqtt-tcp://127.0.0.1:8883
+        server = "mqtt-tcp://broker.emqx.io:1883"
+        
+        # # Protocol version of the bridge.
+        # #
+        # # Value: Enum
+        # # - 5: mqttv5
+        # # - 4: mqttv311
+        # # - 3: mqttv31
+        proto_ver = 4
+        # # The ClientId of a remote bridge.
+        # # Default random string.
+        # #
+        # # Value: String
+        # clientid="bridge_client"
+        
+        # # Ping: interval of a downward bridge.
+        # #
+        # # Value: Duration
+        # # Default: 10 seconds
+        keepalive = 60s
+        # # The Clean start flag of a remote bridge.
+        # #
+        # # Value: boolean
+        # # Default: false
+        # #
+        # # NOTE: Some IoT platforms require clean_start
+        # #       must be set to 'true'
+        clean_start = false
+        # # The username for a remote bridge.
+        # #
+        # # Value: String
+        username = username
+        # # The password for a remote bridge.
+        # #
+        # # Value: String
+        password = passwd
+        
+        ssl {
+            # # enable ssl
+            # # 
+            # # Value: true | false
+            enable = false
+            # # Ssl key password
+            # # String containing the user's password. Only used if the private keyfile
+            # # is password-protected.
+            # #
+            # # Value: String
+            key_password = "yourpass"
+            # # Ssl keyfile
+            # # Path of the file containing the client's private key.
+            # #
+            # # Value: File
+            keyfile = "/etc/certs/key.pem"
+            # # Ssl cert file
+            # # Path of the file containing the client certificate.
+            # #
+            # # Value: File
+            certfile = "/etc/certs/cert.pem"
+            # # Ssl ca cert file
+            # # Path of the file containing the server's root CA certificate.  
+            # # 
+            # # This certificate is used to identify the AWS IoT server and is publicly
+            # # available.
+            # #
+            # # Value: File
+            cacertfile = "/etc/certs/cacert.pem"
+        }
+    }
+}
+
+forward_rules = {
```

Comment:
forward rules should better be shown first

Reply:
move it to the top of file?

Reply:
Yes